cmds/core/shutdown/shutdown.go

```diff
@@ -2,23 +2,31 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// shutdown halts, suspends, or reboots.
+// shutdown halts, suspends, or reboots at a specified time, or immnediately.
```

Comment:
immNediately, litte typo, sorry :)

Reply:
oops. You have no idea how long I looked at that and it looked just a bit off, but I missed it.